### ISSUE TITLE
sway-contrib has changed repo

### ISSUE THREAD

**Issue by seghers** [NixOS/nixpkgs#247231]
### Describe the bug
https://github.com/swaywm/sway/commit/bb91b7f5fa7fddb582b8dddf208cc335d39da9e7

User-contributed scripts are being moved over to this repository:
https://github.com/OctopusET/sway-contrib

I just adopted the use of [nixpkgs-wayland](https://github.com/nix-community/nixpkgs-wayland) and couldn't build grimshot:
``` sh
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/cmj8y55yyiix136ikr6594rf9dkkxibc-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "installPhase" }
installing
install: cannot stat 'contrib/grimshot.1': No such file or directory
```
I could build my nixosconfiguration before when I was using `nixos-unstable`. I guess nixpkgs-wayland is slightly ahead, and we'll have this problem when nixos-unstable updates sway.

### Notify maintainers
@primeos  @Synthetica9 @synthetica @evils

### Metadata
```console
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.42, NixOS, 23.11 (Tapir), 23.11.20230802.66aedfd`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.15.1`
 - nixpkgs: `/nix/store/jm4221xrjmj4kksmn304yrq78vzgjry0-vr3d056k1n1r8fmm9qkz3d1r4cj03bza-source`
```

**Comment by evils** [comment]
i don't think github notifies issue subscribers of linked PRs so:
i've opened https://github.com/NixOS/nixpkgs/pull/247578 to fix this

**Comment by primeos** [comment]
>  I guess nixpkgs-wayland is slightly ahead, and we'll have this problem when nixos-unstable updates sway.

Yes, that's a nixpkgs-wayland problem/regression/bug. Nixpkgs is still fine. We just have to update the package before the next Sway update but it might indeed be best to do this in advance.
